Tests/SwiftParserTest/translated/TrailingClosuresTests.swift

```diff
@@ -145,30 +146,44 @@ final class TrailingClosuresTests: XCTestCase {
     )
   }
 
-  func testTrailingClosures13() {
+  func testTrailingClosures13a() {
```

Comment:
Would be nice to actually give these names rather than just `a` and `b`

Reply:
Yes, that would be nice. But most of the time, I wouldn’t even know how to name these tests (this one is easy, but a lot of the others are not…)